src/stateProviders/provideEntity.ts

```diff
@@ -1,9 +1,16 @@
 import { provideState } from 'freactal';
-import { findIndex, omit, uniq } from 'lodash';
+import { findIndex, isEmpty, omit, uniq } from 'lodash';
 
+import { isGroup, isPolicy } from 'common/associatedUtils';
 import RESOURCE_MAP from 'common/RESOURCE_MAP';
 
 const MAX_ASSOCIATED = 5;
+
+export const getListFunc = (associatedType, parent) =>
+  associatedType === 'permissions' && !isEmpty(parent)
```

Comment:
these magic strings `'permissions'` seem to be scattered around quite a lot, should we extract them to an enum or constants somewhere? seems a bit prone to typos

Reply:
Agree! Will add that